public/externals.json

```diff
@@ -26,6 +26,9 @@
     },
     {
       "path": "https://unpkg.com/vtex-tachyons@1.0.2/tachyons.css"
+    },
+    {
+      "path": "https://unpkg.com/intl@1.2.5/dist/Intl.js"
```

Comment:
Shouldn't we add this do the dynamic poly in render? Recent browsers have it

Reply:
OK, but at least it needs to be loaded for SSR, because nodejs comes only with english locales.